applauncher/004-three-by-three.qml

```diff
@@ -0,0 +1,358 @@
+                currentPressedIndex = index
+            }
+
+            Connections {
+                target: appLauncher
+                function onFakePressedChanged() {
+                    appTitle = model.object.title                   
+                }
+            }
+
+            Item {
+                id: circleWrapper
+
+                width: parent.width * .86
+                height: width
+                anchors {
+                    centerIn: parent
+                    verticalCenterOffset: -parent.height * .08
+                }
+
+                Rectangle {
+                    id: circle
+
+                    anchors.centerIn: parent
+                    width: parent.width
+                    height: width
+                    radius: width/2
+                    color: !pressAndHoldTimer.running ?
+                               launcherItem.pressed | fakePressed ?
+                                           alb.centerColor(launcherModel.get(currentPressedIndex).filePath) :
+                                           "#f4f4f4" : "#f4f4f4"
+                    Behavior on color {
+                        PropertyAnimation { target: circle; property: "color"; duration: 70 }
+                    }
+                }
+            }
+
+            DropShadow {
+                anchors.fill: circleWrapper
+                horizontalOffset: 0
+                verticalOffset: 0
+                radius: 8.0
+                samples: 17
+                color: "#80000000"
+                source: circleWrapper
+                cached: true
+            }
+
+            Icon {
+                id: icon
+
+                width: circleWrapper.width * .70
+                height: width
+                anchors.centerIn: circleWrapper
+                color: !pressAndHoldTimer.running ?
+                           launcherItem.pressed | fakePressed ?
+                                       "#ffffff" :
+                                       "#555555" : "#555555"
+                name: model.object.iconId === "" ? "ios-help" : model.object.iconId
+                Behavior on color {
+                    PropertyAnimation { target: icon; property: "color"; duration: 70 }
+                }
+            }
+        }
+
+        Component.onCompleted: {
+            launcherCenterColor = alb.centerColor(launcherModel.get(0).filePath);
+            launcherOuterColor = alb.outerColor(launcherModel.get(0).filePath);
+
+            toLeftAllowed = false
+            toRightAllowed = false
+            toBottomAllowed =  Qt.binding(function() { return !atYBeginning })
+            toTopAllowed = Qt.binding(function() { return !atYEnd })
+            forbidTop = Qt.binding(function() { return !atYBeginning })
+            forbidBottom = false
+            forbidLeft = false
+            forbidRight = false
+        }
+
+        onContentYChanged: {
+            var lowerStop = Math.floor(contentY/appsView.height)
```

Comment:
I don’t think this color interpolation system makes sense in a 3x3 list where there isn’t any element in focus. Wdyt?

Reply:
I was thinking to tune it into changing to some sane color change per scrolled line of 3 icons.
I converted this to draft before @Magnefire showed how to "fix" the background color change in the 001- launcher.
So i am going to wait how we decide to handle the exact implematation there and then play with some options here before eventually deciding on no color change like you mentioned.

Reply:
I decided to indeed remove the flatmesh color change since changing them per line felt to much when adding the colored shutters on top. Thnaks @MagneFire for the quick `launcherColorOverride = true` hint.